### DISCUSSION TITLE
BT下载确实存在问题

### DISCUSSION THREAD

**Discussion by jh0072110** [Openwrt-Passwall/openwrt-passwall2#101] [General]
因为很少遇到需要下载的环境，现在才大概做了测试，结论：
1.像之前老哥说的一样，迅雷完全没速度（包括会员）
2.使用qBittorrent速度也极大限制，看起来大多数用户都无法成功连接上（但还有那么点点速度……）
大体上就这样，我关了psw2五分钟就下完了也再没机会测试了……

**Comment by yunxi993** [comment]
你是x86吗？
会不会是路由性能影响

**Reply by jh0072110** [reply]
J4125…应该不是性能瓶颈…

**Reply by yunxi993** [reply]
> J4125…应该不是性能瓶颈…

你是用的VLESS吗.

**Reply by jh0072110** [reply]
嗯哼 vless+grpc 不应该是这个问题吧…

**Reply by yunxi993** [reply]
> 嗯哼 vless+grpc 不应该是这个问题吧…

方便看一下你主页的规则吗.

**Reply by jh0072110** [reply]
<img width="615" alt="image" src="https://user-images.githubusercontent.com/50496854/166293218-152f1dc9-a4b1-45e8-8863-ca17e1693c01.png">

**Reply by yunxi993** [reply]
> <img alt="image" width="615" src="https://user-images.githubusercontent.com/50496854/166293218-152f1dc9-a4b1-45e8-8863-ca17e1693c01.png">

配置也没问题，下载速度没法回答你，我这边宽带光猫导致带宽跑不满，没法测试.

**Comment by DearTanker** [comment]
同样有问题，只要把高级设置里面的 UDP不转发端口 设置为 全部就可以了。
但是貌似会导致 UDP 流量全部不通过规则和代理，这就很蛋疼了。。

**Reply by yangfang043** [reply]
不这样设置，游戏都玩不了。

**Reply by wangdaye078** [reply]
如果我想设置成1000以上端口不转发，那个自定义该怎么填，有人试过没。

**Reply by Deleted-Account** [reply]
1000:65535

**Comment by hkongm** [comment]
不得不加一了，感觉是分流规则里 bittorrent 的识别有问题，BT 直连无效

**Comment by handsomeji** [comment]
可以試試最新源代碼，性能大優化~~~
